src/Body/PlainText.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Http\Client\Body;
+
+use Http\Client\Body;
+
+/**
+ * Plain text body
+ *
+ * @author Márk Sági-Kazár <mark.sagikazar@gmail.com>
+ */
+class PlainText implements Body
+{
+    /**
+     * @var string
+     */
+    protected $string;
+
+    /**
+     * @param string $string
+     */
+    public function __construct($string)
+    {
+        $this->string = $string;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getContentHeaders()
+    {
+        return [
+            'Content-Type' => 'text/plain',
```

Comment:
why do we need a special body type for this? should we not simply pass the content type header and the body as a string in this case? if a body class helps a lot, the content type should not be hardcoded but we should have a TextBody or StringBody something like that with a second constructor argument for the Content-Type